### PR TITLE
KHWE-8: Implement the about view

### PR DESCRIPTION
**Pull request by lubovgribiniyk**
joxi_screenshot_1540834151176

### CODE REVIEW

**Comment by ihor4568** [comment]
Please mount this component inside main component. When I click on about button inside sidebar - it should be displayed

**Comment by ihor4568** [comment]
Please remove lorem ipsum and add some meaningful description

File: src/components/Main/Main.js [hunk 1]
```diff
@@ -24,11 +27,25 @@ export class MainComponent {
     );
   }
 
+  addEventListeners() {
+    this.aboutLink.addEventListener(
```
Comment (ihor4568) [inline src/components/Main/Main.js]: please use event delegation

File: src/components/About/About.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import aboutTemplate from "./About.html";
+
+export class About {
```
Comment (ihor4568) [inline src/components/About/About.js]: component name should be `AboutComponent`

**Comment by ihor4568** [comment]
image

please fix build

**Comment by ihor4568** [comment]
image

please set max width for the text - so it will look nicer on large screens

File: src/components/About/About.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
+<section class="about">
+  <div class="about__content">
+    <h2 class="about__heading mdc-typography--headline4">About</h2>
+    <p class="about__text mdc-typography--body1">
+      This app is for people who really like music. Here it’s easy to find the right music for every moment - just open and listen to your favourite songs.
+      You have a few options to find music on our service. Also you can create your own playlist.
+      Our service is completely free to use.
+    </p>
+    <h4 class="about__subtitle mdc-typography--headline6">It is developed by:</h4>
+    <div class="about__links">
```
Comment (ihor4568) [inline src/components/About/About.html:10]: you can use loops or foreach here

File: src/components/Main/Main.js (hunk 1 above)
Comment (lubovgribiniyk) [reply src/components/Main/Main.js]: fixed

File: src/components/About/About.js (hunk 2 above)
Comment (lubovgribiniyk) [reply src/components/About/About.js]: fixed

File: src/components/About/About.html (hunk 3 above)
Comment (lubovgribiniyk) [reply src/components/About/About.html:10]: fixed

**Comment by lubovgribiniyk** [comment]
All comments were fixed.
Build works as expected on my computer. It seems that you removed Page.scss.

File: src/components/Main/Main.js [hunk 4]
```diff
@@ -24,11 +27,24 @@ export class MainComponent {
...
   }
 
+  addEventListeners() {
+    this.sidebarList.addEventListener("click", this.handleListClick.bind(this));
+  }
+
+  handleListClick(e) {
+    e.preventDefault();
+    if (e.target.closest(".main__about-link")) {
+      this.activeView = new AboutComponent(this.mainPoint);
```
Comment (ihor4568) [inline src/components/Main/Main.js]: you could initialize you component in `mountChildren` and then just call `mount` here. There is no need for active view currently

File: src/components/About/About.html [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+      <% ["WOLFRIEND", "lubovgribiniyk", "yevheniiIvanise", "pavelbaranchuk",
```
Comment (ihor4568) [inline src/components/About/About.html]: please move this array to js code

**Review by ihor4568 — APPROVED** [review]

**Pull request merged**